### ISSUE TITLE
Unable to change password, when no password is set

### ISSUE THREAD

**Issue by nebulade** [MediaBrowser/Emby#3465]
Emby version: 3.6.0.60-beta (from the release repo)

Attempting to change the user password, for a user currently without password fails and results in:
```
2018-11-04 18:47:08.513 Info HttpServer: HTTP POST http://kino.nebulon.de:8096/emby/Users/39394e9b0cd548bfa24fdc8dce66bab6/Password. UserAgent: Mozilla/5.0 (X11; Linux x86_64; rv:63.0) Gecko/20100101 Firefox/63.0
2018-11-04 18:47:08.516 Error UserManager: Error authenticating with provider Default
```
This appears to be happening if Firefox detects the hidden input field for the current password ( https://github.com/MediaBrowser/Emby/blob/dffad1c52cbc73336fbbcb91894a0f0220c4c3fb/MediaBrowser.WebDashboard/dashboard-ui/userpassword.html#L29 ) and autofills it. In such a case, it is impossible for the user to see with what the browser autofilled, thus one cannot provide the current password. 

The `autocomplete="off"` does not work as expected at least in firefox and a common solution for this is to add yet another hidden password input field as the first input field in the form.

More can be read on this at https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#The_autocomplete_attribute_and_login_fields

I would have create a  pull request with a fix, however I don't have Visual Studio and thus cannot really build the product as such.

**Comment by LukePulverenti** [comment]
Thanks !

**Comment by LukePulverenti** [comment]
Resolved for Emby Server 3.6.
